RxJavaTemplate/app/src/staging/java/co/nimblehq/rxjava/IdlingResource.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.rxjava
+
+import androidx.test.espresso.idling.CountingIdlingResource
+
+object IdlingResource {
```

Comment:
I think if we could get rid of duplicated on `staging` and `production` by using `sourceSets`